lib/services/local/users.go

```diff
@@ -907,6 +907,16 @@ func (s *IdentityService) GetWebauthnSessionData(ctx context.Context, user, sess
 	if err != nil {
 		return nil, trace.Wrap(err)
 	}
+
+	if s.Clock().Now().UTC().After(item.Expires) {
```

Comment:
is there a point to converting now to UTC?

Reply:
Another Q: is Expires guaranteed to be UTC? (I would guess so, but worthy asking.)

Reply:
Bump?

Reply:
It should be UTC, but go accounts for timezones when comparing times anyways.